gnosis/eth/ethereum_network.py

```diff
@@ -307,7 +307,7 @@ class EthereumNetwork(Enum):
     FREIGHT_TRUST_NETWORK = 211
     PERMISSION = 222
     SUR_BLOCKCHAIN_NETWORK = 262
-    KCC_MAINET = 321
+    KCC = 321
```

Comment:
I would prefer KCC_MAINNET because there is a KCC_TESTNET.

Reply:
We use the name you used for chainlist, we are not changing it: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-321.json